<launch>
  <node pkg="see_ego_motion_ukf" launch-prefix="gdb -ex run --args" type="see_ego_motion_ukf" name="see_ego_motion_ukf" output="screen"></node>
</launch>
